tests/basic_commands_test.go

```diff
@@ -0,0 +1,56 @@
+// basic_commands_test.go
+package tests
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/rs/zerolog/log"
+)
+
+func runL2CommandAndGetOutput(t *testing.T, cmdArgs ...string) string {
+
+	// Get the full path to the l2 binary
+	l2BinPath := "../build/l2"
```

Comment:
How do we ensure check is happening on the latest binary? Any mechanism to avoid the error?

Reply:
Yes every test will build a new binary before executing the tests
![image](https://github.com/HexmosTech/Lama2/assets/66487268/199a72a1-6982-415f-a78c-fb0c17845e35)

Reply:
Locally when we run make test it builds new binary
![image](https://github.com/HexmosTech/Lama2/assets/66487268/0d583712-f49c-44df-a9e0-b4557c5efac8)